dataclient/dataclient.go

```diff
@@ -0,0 +1,60 @@
+package dataclient
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/zalando-incubator/skrop/filters"
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/eskipfile"
+	"github.com/zalando/skipper/routing"
+)
+
+type skropDataClient struct {
+	fileName string
+	prepend  *eskip.Filter
+	append   *eskip.Filter
+}
+
+func NewSkropDataClient(eskipFile string) routing.DataClient {
+
+	emptyArgs := make([]interface{}, 0)
+
+	pre := &eskip.Filter{
+		Name: filters.NewFinalizeResponse().Name(),
```

Comment:
you could just define two constants in the filters package: `SetupResponse` and `FinalizeResponse` because calling the NewX() only to take the name is not very nice, i think.